content/articles/a-table.md

```diff
@@ -0,0 +1,21 @@
+---
+cconlynote: ""
```

Comment:
Surprised there is no article in the static site data with a CC-only note. Implies there is no test against a real article?

Reply:
was manual test - have added here - but not for the test - there is a test with test. data inside the standard article tests, ensuring it renders on the page